iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/receiver/protocol/thrift/IoTDBDataNodeReceiver.java

```diff
@@ -635,6 +711,20 @@ private static TSStatus executeStatement(final Statement statement) {
         .status;
   }
 
+  private static TSStatus executeStatementForTableModel(Statement statement) {
+    return Coordinator.getInstance()
+        .executeForTableModel(
```

Comment:
Executing a tree-model statement for table model will call statement.toRelationalStatement() (in file Statement.java, line 71). please make sure it won't throw UnsupportedOperationException here.

Reply:
@DanielWang2035 Thanks, bro. @luoluoyuyu will resolve this in the following PR...